packages/turf-clusters-distance/index.js

```diff
@@ -25,17 +25,12 @@ module.exports = function (points, maxDistance) {
     collectionOf(points, 'Point', 'Input must contain Points');
 
     // Create index
-    const load = points.features.map(function (point, index) {
```

Comment:
👍 Love it! I think my first approached used the `id` fields to do the matching, however I might of dropped that workflow mid way and forgot to remove the `.map()`.

@stebogit Good catch!